scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/remotes/postgres/GetGoalSatisfactionAction.java

```diff
@@ -0,0 +1,47 @@
+package gov.nasa.jpl.aerie.scheduler.server.remotes.postgres;
+
+import gov.nasa.jpl.aerie.scheduler.server.models.GoalId;
+import gov.nasa.jpl.aerie.scheduler.server.services.ScheduleResults;
+import org.intellij.lang.annotations.Language;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+
+/*package-local*/ final class GetGoalSatisfactionAction implements AutoCloseable {
+  private final static @Language("SQL") String sql = """
+    select
+      goal.goal_id,
+      goal.satisfied
+    from schedulign_goal_analysis as goal
```

Comment:
```suggestion
    from scheduling_goal_analysis as goal
```

Reply:
Are you having trouble finding time to hangout with friends? Does it constantly seem like every time you have a free moment everyone you know is busy? If you're tired of this problem, then it's time to align! Align you schedules with SCHEDULIGN!

Reply:
report spam

unsubscribe

Reply:
Fixed!